serve.go

```diff
@@ -90,13 +90,25 @@ var serveCmd = &cobra.Command{
 
 						//nolint:gosec
 						rand := rand.Int63n(maxNumber)
-						tempFile := filepath.Join(os.TempDir(), fmt.Sprintf("vhs-%d.gif", rand))
+						tempFile := filepath.Join(os.TempDir(), fmt.Sprintf("vhs-%d", rand))
 
 						err = Evaluate(cmd.Context(), b.String(), s.Stderr(), func(v *VHS) {
-							v.Options.Video.Output.GIF = tempFile
-							// Disable generating MP4 & WebM.
-							v.Options.Video.Output.MP4 = ""
-							v.Options.Video.Output.WebM = ""
+							var gif, mp4, webm string
+							switch {
+							case v.Options.Video.Output.MP4 != "":
+								tempFile += ".mp4"
+								mp4 = tempFile
+							case v.Options.Video.Output.WebM != "":
+								tempFile += ".webm"
+								webm = tempFile
+							default:
+								tempFile += ".gif"
+								gif = tempFile
+
```

Comment:
```suggestion
							case v.Options.Video.Output.MP4 != "":
								mp4 = tempFile + ".mp4"
							case v.Options.Video.Output.WebM != "":
								webm = tempFile + ".webm"
							default:
								gif = tempFile + ".gif"
```

Reply:
I would need to try this locally, but iirc I tried this initially and `ffmpeg` complained (at least about gifs) because the output file had an incorrect extension.

Reply:
Ah wait, I need to revert this. 
`tempFile` is still used when cleaning up the temporary file